src/state/migrations/clearOpportunities.ts

```diff
@@ -1,11 +1,11 @@
-import type { ReduxState } from 'state/reducer'
+import type { PersistPartial } from 'redux-persist/es/persistReducer'
+import type { OpportunitiesState } from 'state/slices/opportunitiesSlice/types'
 
 import { initialState } from '../slices/opportunitiesSlice/opportunitiesSlice'
 
-export const clearOpportunities = (state: ReduxState): ReduxState => {
+export const clearOpportunities = (
+  _state: OpportunitiesState,
+): OpportunitiesState & PersistPartial => {
   // Migration to clear opportunitiesApi and opportunitiesApi state
-  return {
-    ...state,
-    opportunities: initialState,
-  }
+  return initialState as OpportunitiesState & PersistPartial
```

Comment:
[self note]: Test individual migrations to ensure these changes to what's intended (i.e not nuking all cache)